ddev/ddev
# docs: show a warning banner for the latest docs (readthedocs+mkdocs)

## The Issue

- #5832

## Alternative

- #5859

## How This PR Solves The Issue

Adds a warning banner.

Resources used in making this:
- https://docs.readthedocs.io/en/stable/reference/environment-variables.html#envvar-READTHEDOCS_VERSION_NAME
- https://docs.readthedocs.io/en/stable/versions.html#version-warning
- https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/?h=outdated#version-warning
- https://github.com/readthedocs/readthedocs.org/issues/8529

Fixes mkdocs warnings on build.

## Manual Testing Instructions

See https://ddev--5855.org.readthedocs.build/en/5855/

![image](https://github.com/ddev/ddev/assets/24270994/418de177-45ed-4223-83a6-8f4da57475b7)

## Automated Testing Overview



## Related Issue Link(s)

## Release/Deployment Notes



## Alternative

As an alternative we could use this https://docs.readthedocs.io/en/stable/versions.html#version-warning

![image](https://github.com/ddev/ddev/assets/24270994/d9ac2f3a-4b8b-4e93-baad-b01a64057ce8)

But it appears over and over again when you navigate the latest docs, so I decided not to use it (disabled it here https://beta.readthedocs.org/dashboard/ddev/addons/edit/).
